packages/runtime/src/index.ts

```diff
@@ -155,7 +155,7 @@ const plugin: NetlifyPlugin = {
     const apiRoutes = await getExtendedApiRouteConfigs(publish, appDir)
 
     await generateFunctions(constants, appDir, apiRoutes)
-    await generatePagesResolver({ target, constants })
+    await generatePagesResolver({ constants })
```

Comment:
[dust] Since there's only one property in the object being destructured, consider passing `constants` as the single argument.